src/Hero.js

```diff
@@ -7,24 +7,24 @@ import IndexGrid from './IndexGrid'
 
 export default function Hero() {
   return (
-    <Box pt={[4, 2, 3]}>
-      <IndexGrid my={[3, 3, 6]} flexDirection={[`row`, `row`, `row-reverse`]} alignItems="end">
-        <FlexItem px={4} width={[10 / 12, 8 / 12, 6 / 12, 5 / 12]} mx={'auto'}>
+    <Box pt={[4, 2, 5]}>
+      <IndexGrid my={[6, 6, 12]} flexDirection={[`row`, `row`, `row-reverse`]} alignItems="end">
```

Comment:
These all can be regular single quotes, surprised the linter didn't pick that up! Looks like it was like that before though so not a big deal to change